src/ui/BibleReferenceSettingTab.ts

```diff
@@ -140,86 +142,55 @@ export class BibleReferenceSettingTab extends PluginSettingTab {
-        toggle
-          .setValue(!!this.plugin.settings?.chapterTagging)
-          .onChange((value) => {
-            this.plugin.settings.chapterTagging = value
-            this.plugin.saveData(this.plugin.settings)
+      .setName('Callout Default Folding')
+      .setDesc('Sets the state of generated callouts upon opening a note')
+      .addDropdown((dropdown: DropdownComponent) => {
+        CalloutFoldFormatCollection.forEach(({ name, description }) => {
+          dropdown.addOption(name, description)
+        });
+        dropdown
+          .setValue(
+            this.plugin.settings.calloutDefaultFold ?? CalloutFoldFormat.NoFold
+          )
+          .onChange(async (value) => {
+            this.plugin.settings.calloutDefaultFold = value as CalloutFoldFormat
+            console.debug('Callout Default Folding To: ' + value)
+            await this.plugin.saveSettings()
+            new Notice('Callout Default Folding Updated')
           })
-      )
+      })
   }
 
-  SetUpBookBacklinking = (containerEl: HTMLElement): void => {
+  SetUpHeaderFormatOptions = (containerEl: HTMLElement): void => {
     new Setting(containerEl)
-      .setName('Create Book Backlink')
-      .setDesc('Makes [[{book}]] link')
-      .addToggle((toggle) =>
-        toggle
-          .setValue(!!this.plugin.settings?.bookBacklinking)
-          .onChange((value) => {
-            this.plugin.settings.bookBacklinking = value
-            this.plugin.saveData(this.plugin.settings)
+      .setName('Header Formatting String')
+      .setDesc('Describes how to format the header of the callout.')
+      .addTextArea((text) => 
+        text
+          .setPlaceholder('')
+          .setValue(this.plugin.settings.headFormatString)
+          .onChange(async (value) => {
+            this.plugin.settings.headFormatString = value;
+            console.debug('Header Format String To:' + value)
+            await this.plugin.saveSettings()
           })
       )
   }
 
-  SetUpChapterBacklinking = (containerEl: HTMLElement): void => {
+  SetUpFooterFormatOptions = (containerEl: HTMLElement): void => {
     new Setting(containerEl)
-      .setName('Create Chapter Backlink')
-      .setDesc('Makes [[{book_chapter}]] link')
-      .addToggle((toggle) =>
-        toggle
-          .setValue(!!this.plugin.settings?.chapterBacklinking)
-          .onChange((value) => {
-            this.plugin.settings.chapterBacklinking = value
-            this.plugin.saveData(this.plugin.settings)
+      .setName('Footer Formatting String')
+      .setDesc('Describes how to format the footer of the callout.')
+      .addTextArea((text) => 
+        text
+          .setPlaceholder('')
+          .setValue(this.plugin.settings.tailFormatString)
+          .onChange(async (value) => {
+            this.plugin.settings.tailFormatString = value;
+            console.debug('Footer Format String To:' + value)
+            await this.plugin.saveSettings()
           })
       )
   }
```

Comment:
If we want to have a template, I believe template should contain 

- header
- content
- footer

For a more industry standard way of template, We should not have a template for header, then a template for footer, so this should be one text area. 

{{[!Bible]}}

{{Content}}

{{Footer}}